includes/class-sensei-utils.php

```diff
@@ -2699,6 +2699,16 @@ public static function get_featured_video_thumbnail_url( $post_id ) {
 	public static function is_atomic_platform(): bool {
 		return defined( 'ATOMIC_SITE_ID' ) && ATOMIC_SITE_ID && defined( 'ATOMIC_CLIENT_ID' ) && ATOMIC_CLIENT_ID;
 	}
+
+	/**
+	 * Tells if the current site is hosted in wordpress.com and the
+	 * plan includes an active woothemes-sensei paid plan.
+	 */
+	public static function has_wpcom_subscription(): bool {
+		$subscriptions = get_option( 'wpcom_active_subscriptions', [] );
+
+		return isset( $subscriptions['woothemes-sensei'] );
```

Comment:
@jom Here, I'm using the exact same code you mentioned on the issue, but, thinking again, I believe we'll need to have this dynamically set, maybe checking for an array of products instead of just checking out for `woothemes-sensei`? WDYT?

Reply:
@fjorgemota Yes, here we'll need to check all three plugins. However, right now, Sensei Blocks doesn't use this provider for the contact form:

https://github.com/Automattic/sensei-pro/blob/a8379d8e7d34c22b8cacd0792c1325aeaaff718d/modules/sensei-blocks-home/includes/providers/class-help.php#L32

I'm not sure if it makes sense to hardcode them into Sensei core or to provide a filter and have the plugins inject themselves into it.

Reply:
Hey, I decided to provide a filter and have the plugins inject themselves into it. Changed in [09a52a3](https://github.com/Automattic/sensei/pull/6407/commits/09a52a39db4380c94a9ed3371fd22d6f5bc92f72). Thanks!